### PR TITLE
Clean gh-pages

### PR DESCRIPTION
**Pull request by dgarcia360**
This commit cleans the gh-pages branch. Before adding javadoc generated docs (#41), it would be great to merge this PR to avoid conflicts with the new documentation system.

### CODE REVIEW

**Review by haaawk — APPROVED** [review]
LGTM

**Comment by haaawk** [comment]
@lauranovich please review. I will merge after you approve the change.

**Review by lauranovich — APPROVED** [review]

**Comment by lauranovich** [comment]
@haaawk approved

**Pull request merged**
